blog/test.py

```diff
@@ -1,6 +1,8 @@
+import os
 from unittest import TestCase
 
 from .models import Article
+from bin.fixtures import create_article
```

Comment:
A lib or test should not call a bin script.
You could simply create a _dummy_ article. See below.